eland/ml/exporters/_sklearn_deserializers.py

```diff
@@ -97,18 +97,23 @@ def __init__(
                     impurity[i],
                     n_node_samples[i],
                     weighted_n_node_samples[i],
+                    True,
                 )
                 for i in range(node_count)
             ],
-            dtype=[
-                ("left_child", "<i8"),
-                ("right_child", "<i8"),
-                ("feature", "<i8"),
-                ("threshold", "<f8"),
-                ("impurity", "<f8"),
-                ("n_node_samples", "<i8"),
-                ("weighted_n_node_samples", "<f8"),
-            ],
+            dtype={
```

Comment:
We should update the scikit learn version in `requirements-dev.txt` and `setup.py` otherwise these tests will fail for anyone on an older version

Currently it is:
```
scikit-learn>=0.22.1,<2
```

Suggest:
```
scikit-learn>=1.3,<2
```